### DISCUSSION TITLE
v1.2断流严重

### DISCUSSION THREAD

**Discussion by quango-web** [XTLS/Xray-core#126] [Performance]
客户端passwall集成xray1.2 ，gcp-vps- debian10 ，通过脚本安装vless+tcp+xtls并更新为xray1.2，连接后很快断流，TV油管奈非显示无网络

**Comment by RPRX** [comment]
目测是 TPROXY UDP 与 iptables 一些规则冲突的问题

**Comment by feiyue789** [comment]
用的机场节点，之前我passwall都是默认xray内核没问题，昨天更新1.2.0之后电视上面就打不开奈飞，手机上面正常，换成1.1.5之后没问题了

**Comment by RPRX** [comment]
稍后我会研究下 PassWall 的 iptables

**Comment by RPRX** [comment]
对了，麻烦贴一下 PassWall 生成的配置（而且这个为什么要发到 discussion，无法转成 issue

**Comment by RPRX** [comment]
麻烦标明你们用的是什么架构，我好发测试版

**Reply by quango-web** [reply]
固件：https://github.com/LewiVir/NanoPi-R4S
版本 | OpenWrt R20.12.12 / LuCI Master (git-20.343.54716-6fc079f)
软路由：NanoPi-R4S
架构：ARMv8 Processor x 6
内核版本 | 5.4.85

**Reply by quango-web** [reply]
passwall: Xray 版本【 1.2.0 】, V2ray 版本【 4.33.0 】
 配置在v2ray-vless则正常

**Reply by RPRX** [reply]
@quango-web 

请问 1.1.5 版本有此情况吗

**Reply by quango-web** [reply]
只要保持客户端1.15就不会

**Reply by quango-web** [reply]
vps-debian10-xray1.2+客户端1.15基本正常；
vps-debian10-xray1.2+客户端1.2完全不过流量，有种ip被墙的感觉

**Reply by RPRX** [reply]
开了 IPv6 吗

**Reply by feiyue789** [reply]
> 麻烦标明你们用的是什么架构，我好发测试版

我用的x86软路由

**Reply by quango-web** [reply]
没开IPv6

**Comment by shenlijun** [comment]
120比115cpu占用要高一点点
性能不是特别强的设备上，100%占用，会有几十兆的速度差距

**Reply by badO1a5A90** [reply]
120和115应该没有任何性能上的改变,你的使用场景和配置有变化吗?

**Comment by RPRX** [comment]
不确定 passwall 最新版有没有修复，可以试一下，目前 ssrp 没有这个问题，也可以试试

**Comment by RPRX** [comment]
@quango-web @feiyue789 试试这个 commit https://github.com/XTLS/Xray-core/commit/ae98dc75cf30451d3f29684977ff7324e0fc26e4

**Comment by RPRX** [comment]
试试 https://github.com/xiaorouji/openwrt-passwall/issues/879#issuecomment-763509357
